### PR TITLE
[RF] POM > AT_16.03_001 | Delete the Multibranch Pipeline using dropdown menu

### PR DESCRIPTION
**Pull request by SvetlanaG1011**
https://trello.com/c/llPrS01k/1593-rf-pom-at1603001-delete-the-multibranch-pipeline-using-dropdown-menu

### CODE REVIEW

**Comment by Rom-T** [comment]
Pull the data from main and everything will pass

**Comment by SvetlanaG1011** [comment]
> Pull the data from main and everything will pass



> Pull the data from main and everything will pass



> Pull the data from main and everything will pass

Thank you!

**Review by AramH20 — APPROVED** [review]

**Review by engineeryulia — APPROVED** [review]
Great job!

File: cypress/pageObjects/HomePage.js [hunk 1]
```diff
@@ -13,6 +15,9 @@ class HomePage {
     getProjectNameLink = () => cy.get('a[href*="job/"]');
     getPageBody = () => cy.get("#page-body");
     getMainPanel = () => cy.get('#main-panel');
+    getProjectDrpDwnBtn = () => cy.get('table#projectstatus button.jenkins-menu-dropdown-chevron');
+    getDeleteMultiBrPipelineLink = () => cy.get('a[href*="/delete"]');
+    getSearchBox = () => cy.get('#search-box');
```
Comment (Rom-T) [inline cypress/pageObjects/HomePage.js:20]: This element is on the page HeaderAndFooter and it's already created

File: cypress/pageObjects/HomePage.js [hunk 2]
```diff
@@ -43,5 +48,26 @@ class HomePage {
...
+
+  hoverProjectNameLink() {
+    this.getProjectNameLink().realHover();
+    return this;
+  }
+
+  clickProjectDrpDwnBtn() {
+    this.getProjectDrpDwnBtn().click({force: true});
+    return this;
+  }
```
[the remark this answers is not in the document]
Comment (AlexSQE) [reply cypress/pageObjects/HomePage.js:60]: Можно сделать без {force: true}:
    clickProjectDrpDwnBtn() {
        this.getProjectDrpDwnBtn().realHover().click();
    return this;

**Pull request merged**